_posts/2021-05-22-fluxv2-openfaas.md

```diff
@@ -0,0 +1,332 @@
+---
+title: "See how Flux's Git Ops ToolKit can simplify delivery"
+description: "See how Flux can simplify serverless deployments to OpenFaaS"
```

Comment:
Same as above. 

"The original version of Flux is widely used, but in maintenance mode. Learn how to upgrade to Flux v2."